src/prefect/client/orion.py

```diff
@@ -185,9 +215,8 @@ async def api_healthcheck(self) -> Optional[Exception]:
         If successful, returns `None`.
         """
         try:
-            with anyio.fail_after(10):
-                await self._client.get("/health")
-                return None
+            await self._client.get("/health")
```

Comment:
I think it's a bad idea to handle this ourselves with `fail_after`. Best case scenario is we fail a bit faster. Worst case (and I think more common) scenario is we disrupt `httpx`'s error handling

Reply:
+1

Reply:
I agree